packages/wallets/walletconnect/src/index.ts

```diff
@@ -1 +1 @@
-export * from './adapter.js';
```

Comment:
is this breaking anybody?

Reply:
Our package is a replacement of that file, has the same exports.